src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.config;
+
+import java.util.Properties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
Does it work without 

`@Override`
 `protected void configure(AuthenticationManagerBuilder auth) throws Exception ` ?

Reply:
I think yes, I can authenticate and throw http methods. How can I check if it doesn't work?